<launch>

    <!-- gst_camera nodelet -->
    <node name="camera_node"
          pkg="easyscan" type="camera_node"
          args="">
          <remap from="camera/image1" to="camera/image"/>
    </node>

</launch>
